static/app/types/core.tsx

```diff
@@ -118,10 +118,11 @@ export enum Outcome {
   ACCEPTED = 'accepted',
   FILTERED = 'filtered',
   INVALID = 'invalid',
-  DROPPED = 'dropped', // this is not a real outcome coming from the server
+  ABUSE = 'abuse',
   RATE_LIMITED = 'rate_limited',
   CLIENT_DISCARD = 'client_discard',
   CARDINALITY_LIMITED = 'cardinality_limited',
+  DROPPED = 'dropped', // this is not a real outcome coming from the server
```

Comment:
@priscilawebdev do we still need `Outcome.DROPPED`?

Reply:
we are still using it in getsentry, but once we update the chart there, we can drop the "dropped" 😄